frame/nomination-pools/src/tests.rs

```diff
@@ -3688,3 +4024,311 @@ mod update_roles {
 		})
 	}
 }
+
+mod reward_counter_precision {
+	use sp_runtime::FixedU128;
+
+	// TODO: SR-LABS must audit these tests. Only remove once approved.
```

Comment:
Was this taken into account @jakoblell ?

Reply:
I believe they have been, but will wait for @jakoblell to confirm.

Reply:
Yes the tests have been audited.